todo_app/data/api.py

```diff
@@ -0,0 +1,97 @@
+from configparser import ConfigParser
+from .boards import *
+from todo_app.config import *
+import os
+import requests
+
+
+class TrelloAPI():
+    TRELLO_URL = 'https://api.trello.com/1'
+    
+    def __init__(self):
+        self.key_and_token = {'key':APIConfig.KEY,'token':APIConfig.TOKEN}
+    
+    def get_boards(self):
+        boards_url = self.TRELLO_URL + '/members/me/boards'
+        key_and_token = self.key_and_token
+        headers = {"Accept": "application/json"}
+        arguments = {'fields': 'name', 'lists': 'open'}
+        response = requests.get(boards_url, params=key_and_token, data=arguments)
+        json_response = response.json()
+        return json_response
+    
+    
+    def get_list_of_items(self):
+        boardId = "oIJEvych"
+        cards_url = self.TRELLO_URL + '/boards/' + boardId + '/cards/'
+        key_and_token = self.key_and_token
+        headers = {"Accept": "application/json"}
+        arguments = {'fields': 'name', 'lists': 'open'}
+        response = requests.get(cards_url, params=key_and_token, data=arguments)
+        json_response = response.json()
+    
+        return json_response
+    
+    def get_lists(self): 
+        lists_on_boards = self.get_list_of_items()
+        boardId = "oIJEvych"
```

Comment:
We should move this board id and the list ids below into config/environment variables too - otherwise the app can only work with a single trello board